### DISCUSSION TITLE
How to change the text of "You say," and "says,"

### DISCUSSION THREAD

**Discussion by TermCIC** [evennia/evennia#3215] [Evennia Questions] [answered]
I am coding a MUD in Chinese, and i don't know how to change the text of "You say," and "says," 
i have tried modify the commands in Evennia, it does not work. I also tried using locales/zh, also doesnt work.

**Comment by Griatch** [comment, marked as answer]
You control this from the `Character` class, especially by overriding the `at_say` method. If you investigate the [say command](https://www.evennia.com/docs/latest/api/evennia.commands.default.general.html#evennia.commands.default.general.CmdSay) you'll find that it is calling this method on the caller to determine how they would say the message and where to send it to. This is where the `say/says` strings are added.

[Here's where the `at_say` method is covered in the docs](https://www.evennia.com/docs/latest/api/evennia.objects.objects.html#evennia.objects.objects.DefaultObject.at_say). 

Hope that helps!
